src/main/frontend/handler/page.cljs

```diff
@@ -577,10 +577,17 @@
                      (util/get-relative-path edit-block-file-path ref-file-path)
                      page)
         (let [journal? (date/valid-journal-title? page)
-              ref-file-path (str (get-directory journal?)
-                                 "/"
-                                 (get-file-name journal? page)
-                                 ".org")]
+              ref-file-path (str
```

Comment:
Avoid using `path/join` here, since it replace `file:///` to `file:/` which breaks mobile path.